qa/acceptance/spec/spec_helper.rb

```diff
@@ -56,3 +56,14 @@
 SpecsHelper.configure(selected_boxes)
 
 puts "[Acceptance specs] running on #{ServiceTester.configuration.hosts}" if !selected_boxes.empty?
+
+def with_running_logstash_service(logstash)
+  begin
+    logstash.start_service
+    Stud.try(40.times, RSpec::Expectations::ExpectationNotMetError) do
+      yield
+    end
```

Comment:
This certainly works for all of the specs you have introduced in this PR, but it may prove to be a mismatch when we introduce other specs that check other things with the running service.

It may be helpful to only yield _once_ when we know that logstash is actually running. I know this double-checks in the cases of our current use, but it also ensures that we don't retry _other_ expectations 40x in this helper.

```suggestion
    Stud.try(40.times, RSpec::Expectations::ExpectationNotMetError) do
      expect(logstash).to be_running
    end
    
    yield
```

Reply:
Your suggestion didn't quite work - `expect(logstash).to be_running` doesn't work when a specific `jdk_path` is used, but I've added the equivalent to the helper method